scm-ui/ui-webapp/src/repos/components/ImportFullRepository.tsx

```diff
@@ -42,35 +42,36 @@ const ImportFullRepository: FC<Props> = ({
   setImportPending,
   setImportedRepository,
   nameForm: NameForm,
-  informationForm: InformationForm,
+  informationForm: InformationForm
 }) => {
   const [repo, setRepo] = useState<RepositoryCreation>({
     name: "",
     namespace: "",
     type: repositoryType.name,
     contact: "",
-    description: "",
-    contextEntries: [],
+    description: ""
   });
   const [password, setPassword] = useState("");
   const [valid, setValid] = useState({ namespaceAndName: false, contact: true, file: false });
   const [file, setFile] = useState<File | null>(null);
   const [t] = useTranslation("repos");
   const { importFullRepository, importedRepository, isLoading, error } = useImportFullRepository(repositoryType);
 
-  useEffect(() => setRepo({ ...repo, type: repositoryType.name }), [repositoryType]);
-  useEffect(() => setImportPending(isLoading), [isLoading]);
+  useEffect(() => setImportPending(isLoading), [isLoading, setImportPending]);
   useEffect(() => {
     if (importedRepository) {
       setImportedRepository(importedRepository);
     }
-  }, [importedRepository]);
+  }, [importedRepository, setImportedRepository]);
 
-  const isValid = () => Object.values(valid).every((v) => v);
+  const isValid = () => Object.values(valid).every(v => v);
 
   const submit = (event: FormEvent<HTMLFormElement>) => {
     event.preventDefault();
-    importFullRepository(repo, file!, password);
+    if (!file) {
```

Comment:
Shouldn't there be error handling or something like this here ? I know this shouldn't ever happen but in the event that it does, the user does not get any feedback.